src/components/NoData.tsx

```diff
@@ -4,7 +4,7 @@ import '@styles/NoData.styles.scss'
 export const NoData = () => {
   return (
     <div className='no-data-container'>
-      <img className='cat-img' src={CryingCat} />
+      <img className='cat-img' alt='cat crying' src={CryingCat} />
```

Comment:
```
<img className='cat-img' alt='Loading' src={CryingCat} />
```

I prefer to us `Loading` here instead of describing the image itself, this is more meaningful.